### PR TITLE
change up prompt with traditional nushell colors

### PR DESCRIPTION
**Pull request by fdncred**
This is totally up for debate. Just looking for something less boring, and less ugly blue. :) I'm up for just about any colors other than everything being the same boring blue.

image
that's in windows - left prompt is `green bold` (like old nushell), indicator is `cyan bold` (like old nushell), right prompt is `dark orange`. left prompt replaces `home dir` with `~` if appropriate and also changes `\\` to `/`.

### CODE REVIEW

File: src/prompt/default.rs [hunk 1]
```diff
@@ -77,7 +77,20 @@ impl DefaultPrompt {
 
 fn get_working_dir() -> Result<String, std::io::Error> {
     let path = env::current_dir()?;
-    Ok(path.display().to_string())
+    let path_str = path.display().to_string();
+    let homedir: String = match env::var("USERPROFILE") {
+        Ok(win_home) => win_home,
+        Err(_) => match env::var("HOME") {
+            Ok(maclin_home) => maclin_home,
+            Err(_) => path_str.clone(),
```
Comment (sholderbach) [inline src/prompt/default.rs:85]: This would in the failure case always result in `~`?

File: src/prompt/default.rs [hunk 2]
```diff
@@ -77,7 +77,20 @@ impl DefaultPrompt {
...
+        },
+    };
+    let new_path = if path_str != homedir {
+        path_str.replace(&homedir, "~").replace("\\", "/")
```
Comment (sholderbach) [inline src/prompt/default.rs:89]: Let the Windows users live in their world with the backwards slashes :P

**Review by sholderbach — COMMENTED** [review]
Like the styling. Do we want to keep `get_prompt_color()` around? Feels like an unnecessary special case

File: src/prompt/default.rs (hunk 2 above)
Comment (fdncred) [reply src/prompt/default.rs:89]: ya, i'm good with whatever.

File: src/prompt/default.rs (hunk 1 above)
Comment (fdncred) [reply src/prompt/default.rs:85]: I tested it, by changing `USERPROFILE` to `USERPROFILE2` which doesn't exist and on WSL I changed `HOME` to `HOME2` which doesn't exist. Help me understand what you mean please.

File: src/prompt/default.rs (hunk 2 above)
Comment (fdncred) [reply src/prompt/default.rs]: removed the replace \ with / part

**Comment by fdncred** [comment]
> Like the styling. Do we want to keep `get_prompt_color()` around? Feels like an unnecessary special case

We're still using `get_prompt_color()` for getting the left prompt color.

**Comment by fdncred** [comment]
confirmed that it looks good in apple's terminal.app.
Screen Shot 2022-04-07 at 6 36 56 PM
that orange may be too much - other suggestions?

**Pull request merged**